packages/datadog-plugin-amqp10/test/integration-test/server.mjs

```diff
@@ -0,0 +1,21 @@
+import 'dd-trace/init.js'
+import amqp from 'amqp10'
+
+const client = new amqp.Client()
+let sender
+
+async function connectToAMQP () {
```

Comment:
This function might as well be inlined, aka pull the contents from the function into the outer scope, since it's only called a single time.